MCprep_addon/spawner/spawn_util.py

```diff
@@ -187,9 +226,11 @@ class ListItemAssets(bpy.types.PropertyGroup):
 	ListMobAssets,
 	ListMeshswapAssets,
 	ListItemAssets,
+        ListModelAssets,
```

Comment:
nit: Spaces vs tabs, surprised it doesn't complain but I suppose any newline/whitespace within lists is technically ok, but better to be consistent.